doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,642 @@
+
+* :ref:`ot_cli_sample` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l10/cpuapp`` board target.
+
+* :ref:`ot_coprocessor_sample` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l10/cpuapp`` and ``nrf54l15dk/nrf54l05/cpuapp`` board targets.
+
+Other samples
+-------------
+
+* :ref:`coremark_sample` sample:
+
+  * Updated:
+
+    * The logging format in the standard logging mode to align it with the format used in the multi-domain logging mode.
+    * Support for alternative configurations to use the :ref:`file suffix feature from Zephyr <app_build_file_suffixes>`.
+      The following file suffixes are supported as alternative configurations:
+
+      * ``flash_and_run``
+      * ``heap_memory``
+      * ``static_memory``
+      * ``multiple_threads``
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+Wi-Fi drivers
+-------------
+
+* Added the :ref:`nrf70_wifi_tx_power_calculation` section to the :ref:`nrf70_wifi` page.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Binary libraries
+----------------
+
+* :ref:`liblwm2m_carrier_readme` library:
+
+  * Updated the :ref:`req_appln_limitations` page to clarify carrier-specific requirements.
+    Added overlay files and documentation to :ref:`serial_lte_modem` application and :ref:`lwm2m_carrier` sample to guide in the correct usage of LwM2M carrier library for SoftBank and LG U+.
+
+Bluetooth libraries and services
+--------------------------------
+
+* Added the :ref:`rreq_readme` and :ref:`rrsp_readme` libraries.
+
+* :ref:`hogp_readme` library:
+
+  * Updated the :c:func:`bt_hogp_rep_read` function to forward the GATT read error code through the registered user callback.
+    This ensures that API user is aware of the error.
+
+Modem libraries
+---------------
+
+* :ref:`modem_key_mgmt` library:
+
+  * Added the CME error code 527 - invalid content.
+  * Updated to handle generic CME errors from all ``AT%CMNG`` commands.
+
+nRF RPC libraries
+-----------------
+
+* Added the :ref:`nrf_rpc_dev_info` library for obtaining information about a device connected through the :ref:`nrfxlib:nrf_rpc`.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`nrf_desktop_config_channel_script` Python script:
+
+  * Added support for pure ED25519 signature (used by nRF54L-based devices that enable MCUboot hardware cryptography).
+    This requires using ``imgtool`` supporting pure ED25519 signature that can be installed from ``sdk-mcuboot`` repository.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a4eda30f5b0cfd0cf15512be9dcd559239dbfc91``, with some |NCS| specific additions.
```

Comment:
The upstream mcuboot SHA should be `e890df7ab975da181a9f3fb3abc470bf935625ab`, @nvlsianpu @nordicjm please double check.

Reply:
Updated.